src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -131,12 +131,10 @@
             div.id = 'Error-div';
```

Comment:
Maybe use a pop up box instead of the error div/Or move it so its in a more visible space?